tests/module_6/test_predict_endpoint.py

```diff
@@ -0,0 +1,29 @@
+from fastapi.testclient import TestClient
+from src.module_6.app import app
+
+client = TestClient(app)
+
+
```

Comment:
This is more like an integration test. You could test `predict` function instead, mocking `feature_store` and `model.predict` to test the two scenarios. That would make it unitary and independent of everything else.